ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -311,7 +311,9 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar, cl
         <GenericModal
           open={!!profileForModal}
           Content={
+            <>
             <Paper className={classes.addProfileModal} >
+            <span onClick={()=> setProfileForModal(undefined)}>Close</span>
```

Comment:
Don't add just close text, take reference from other components you can add cross icon over here

Reply:
ohh you already mentioned in the above checklist 👍

Reply:
Yeah 😄